packages/tasks/src/model-libraries.ts

```diff
@@ -680,6 +680,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path_extension:"safetensors" OR path_extension:"pt"`,
 	},
+	"mochi-1-preview": {
```

Comment:
Just a question: isn't this too specific? Should it just be "mochi" or "mochi-1"?

Reply:
I don't have a good understanding of how libraries work (e.g, how is this linked to the actual huggingface page). I named it after the hugging face repository: https://huggingface.co/genmo/mochi-1-preview. But let me know if it should be called something else.

Reply:
It depend how you think the library will evolve. The name should match the library name from the model card metadata ([here](https://huggingface.co/genmo/mochi-1-preview/blob/8e9673c5349979457e515fddd38911df6b4ca07f/README.md?code=true#L8)) which is currently `library_name: mochi-1-preview`. But it doesn't have to match the model repo id. 

If the goal of the Python library on Github is to support this first `preview` version and then maybe some other versions or finetunes, then I would advice to chose `mochi-1` for instance (as @pcuenca suggested). You can see the library as a family of models, not just one. If we go with `mochi-1-preview`, it means that either all other models will have to be tagged as `mochi-1-preview` or that we will have to add support for a new library tag.

Reply:
Agree with @Wauplin's comments. You can see examples of libraries in this file, they include obvious ones such as `transformers`, but also smaller and more specific ones, for example https://huggingface.co/models?library=depth-anything-v2

Reply:
I see, so this is enabling download counting for an entire library. Which means whatever file path I specify here should be found in all models. To be honest, I don't know if `vae_stats.json` will exist in all possible HuggingFace repositories. But, I guess we can update the elastic search query in the future.

Reply:
Maybe I should rename the library to just "genmo"?

Reply:
Yes! Just `genmo` would be more cleaner and also make this PR forward compatible too.

Reply:
`genmo`'s good in that case!
Since it will be compatible with a large variety of models, it could be nice to have at least a form of similar structure between models. Typically, you could add a `config.json` file with minimal information about the model, its architecture and what it is used for. If all genmo models has one, it might help the library + it will allow us to count the downloads in a consistent way. (I didn't chose the example randomly, `config.json` is actually the default file we are counting downloads from).

But this can be done in a later PR, once you have at least 2 genmo models.

Reply:
Alright, I updated the repository to have the library "genmo": https://huggingface.co/genmo/mochi-1-preview.

I also updated the library name to genmo.
If there are any small nits, do you want to just do the commits and merge? I'm super un-opinionated here.